ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/custom-dns/cucushift-installer-rehearse-aws-ipi-custom-dns-workflow.metadata.json

```diff
@@ -1,5 +1,5 @@
 {
-	"path": "cucushift/installer/rehearse/aws/ipi/user-provisioned-dns/cucushift-installer-rehearse-aws-ipi-user-provisioned-dns-workflow.yaml",
+	"path": "cucushift/installer/rehearse/aws/ipi/custom-dns/cucushift-installer-rehearse-aws-ipi-custom-dns-workflow.yaml",
```

Comment:
On gcp, the workflow folder is still `user-provisioned-dns`, shall we keep them consistent.

Reply:
We can use `custom-dns` for dev ci workflow name.